5.1/base/Gemfile

```diff
@@ -1,21 +1,20 @@
 
 source 'https://rubygems.org'
 
-gem 'rails', '~> 5.1.4'
-gem 'puma', '~> 3.7'
-gem 'redis', '~> 3.2'
-gem 'pg', '~> 0.18.3'
-gem 'bcrypt', '~> 3.1.7'
+gem 'rails', '5.2.1'
```

Comment:
The directory here says 5.1. In that case, I don't think you should change it here. Rather add another directory, I guess?

Reply:
Changed!